src/runtime/virtcontainers/sandbox_test.go

```diff
@@ -1654,3 +1655,50 @@ func TestSandboxHugepageLimit(t *testing.T) {
 	err = s.updateResources(context.Background())
 	assert.NoError(t, err)
 }
+
+func TestPrepareInitdataSquashFsImage(t *testing.T) {
+
+	tests := []struct {
+		name      string
+		imagePath string
+		content   string
+	}{
+		{
+			"create an initdata image",
+			"/tmp/initdata-image-test1",
```

Comment:
Nit: this could use https://pkg.go.dev/testing#T.TempDir.

Reply:
Thanks! I am not familar with golang, let me take a try